internal/tests/testbhs/test_bhs.go

```diff
@@ -1,26 +1,27 @@
-package testpulse
+package testbhs
```

Comment:
I'm not sure about the package name,
maybe `testapp` or `testserver` or 🤷  would be better

Reply:
I updated it to testapp